dump/ruby4.0/dump.sh

```
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+set -eux
+
+ARCH=${1:-}
+if [ -z "$ARCH" ]; then
+  echo "Usage: $0 <architecture>"
+  exit 1
+fi
+fi
```

Shellcheck (0.11.0)

[error] Parsing stopped here. Is this keyword correctly matched up?

(SC1089)